src/llama_recipes/inference/checkpoint_converter_fsdp_hf.py

```diff
@@ -3,14 +3,16 @@
 
 # from accelerate import init_empty_weights, load_checkpoint_and_dispatch
 
-import fire
 import os
 import sys
 import yaml
+import fire
 
 from transformers import AutoTokenizer
 
-from llama_recipes.inference.model_utils import  load_llama_from_config
+from llama_recipes.inference.model_utils import load_llama_from_config
+
+from model_checkpointing import load_sharded_model_single_gpu
```

Comment:
This import will fail as the path is not added sys.path in line 22/24 (which is something we should get rid of in any case). Lets just change it to from llama_recipes.model_checkpointing import load_sharded_model_single_gpu